tests/test_weather_normalization.py

```diff
@@ -83,9 +81,7 @@ def test_weather_retrieval(results_dir, filename):
     and sys.version_info.micro <= 2,
     reason="Skipping Windows and Python <= 3.13.2 due to known bug",
 )
-@pytest.mark.parametrize(
-    "filename", ira_rebate_hpxmls + real_home_hpxmls + ihmh_home_hpxmls, ids=lambda x: x.stem
-)
+@pytest.mark.parametrize("filename", ira_rebate_hpxmls, ids=lambda x: x.stem)
```

Comment:
Reducing the number of files we unit test here. All files will have mini calibrations performed on, so not necessary to test every file here.